stdlib/2and3/difflib.pyi

```diff
@@ -12,7 +12,7 @@ if sys.version_info >= (3,):
     _StrType = Text
 else:
     # Aliases can't point to type vars, so we need to redeclare AnyStr
-    _StrType = TypeVar('_StrType', str, bytes)
+    _StrType = TypeVar('_StrType', Text, bytes)
```

Comment:
Just a minor nit (for future reference): Since this is in the Python 2-only branch, I find using `unicode` directly clearer than `Text`.

Reply:
Noted, thanks!